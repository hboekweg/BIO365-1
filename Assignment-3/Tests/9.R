source("9.R")

smallMammals <- c("gopher", "squirrel", "beaver", "mouse", "squirrel", "squirrel", "mouse", "weasel", "weasel", "squirrel", "bobcat", "gopher")
weights <- c(8.2, 3.1, 13.0, 4.4, 1.9, 2.3, 2.2, 6.5, 5.9, 2.8, 15.3, 7.4)

print(findMaxWeights(smallMammals, weights))