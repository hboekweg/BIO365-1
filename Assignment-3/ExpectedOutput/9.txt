  beaver   bobcat   gopher    mouse squirrel   weasel 
    13.0     15.3      8.2      4.4      3.1      6.5 
